test/algorithms/KirlikSayin.jl

```diff
@@ -0,0 +1,114 @@
+module TestKirlikSayin
+
+using Test
+using JuMP
```

Comment:
This is my fault. I just updated all of the tests to remove JuMP: https://github.com/odow/MOO.jl/pull/7.

I'll show the equivalent below.